js/main.js

```diff
@@ -1,6 +1,73 @@
 const menuIcon = document.querySelector('.menuIcon');
 const nav = document.querySelector('.navList');
 const linkedWord = document.querySelectorAll('.navWord');
+const modal = document.querySelector('.modals');
+const button = document.querySelectorAll('.cardBtn');
+const projectButton = document.querySelector('.seeButton');
+
+const projectCollection = [
```

Comment:
Pop-up on mobile and tablet view does not have `See live` and `See source` buttons. Kindly see the screenshot below and make the pop-up exact same as in the design
![mobile](https://user-images.githubusercontent.com/35504149/137355068-d9e106d3-2cb8-4972-b43d-e2b3aa9a70c7.png)
![tablet](https://user-images.githubusercontent.com/35504149/137355076-5e7bc926-cc87-41f8-a5ea-5f6395d4dcbe.png)
.